rust-1.51/Makefile

```diff
@@ -0,0 +1,74 @@
+GEORUST_ROOT=$(CURDIR)/../..
```

Comment:
I've had some flavor of this makefile floating around uncommitted for a while.

I'm not super proud of it, and if it's controversial, I can leave it out for now.

Reply:
Seems pretty sensible to me.